src/main/kotlin/no/nav/familie/ef/iverksett/tilbakekreving/OpprettTilbakekrevingTask.kt

```diff
@@ -38,41 +38,48 @@ class OpprettTilbakekrevingTask(private val iverksettingRepository: Iverksetting
 
     override fun doTask(task: Task) {
         val behandlingId = UUID.fromString(task.payload)
-        val iverksett = iverksettingRepository.hent(behandlingId)
+        hentIverksettinghvisTilbakekreving( behandlingId)?.let {
+            opprettTilbakekreving(behandlingId, it)
+        }
+    }
 
-        if (!iverksett.vedtak.tilbakekreving.skalTilbakekreves) {
+    private fun hentIverksettinghvisTilbakekreving(behandlingId: UUID): Iverksett? {
+        val iverksett = iverksettingRepository.hent(behandlingId)
+        if (iverksett.behandling.behandlingType == BehandlingType.FØRSTEGANGSBEHANDLING) {
+            logger.warn("Førstegangsbehandling trenger ikke tilbakekreving behandlingId=$behandlingId")
+            return null
+        } else if (!iverksett.vedtak.tilbakekreving.skalTilbakekreves) {
             logger.info("Tilbakekreving ikke valgt for behandlingId=$behandlingId. Oppretter derfor ikke tilbakekrevingsbehandling.")
-            return
+            return null
```

Comment:
For å unngå return null, går det ikke å splitte opp det i en sjekk som kun sjekker om det skal tilbakekreves?
https://nav-it.slack.com/archives/G012YEK9YQ1/p1651501165325649?thread_ts=1651496970.658479&cid=G012YEK9YQ1

Reply:
Ja, det hadde vært best, men litt usikker på om jeg får det til på en veldig god måte. 

1. Jeg vil gjerne unngå å gjøre en ny beregning/simulering hvis vi vet at vi ikke skal opprette (ikke valgt førstegangs osv) 
2. Jeg hadde lyst til å slippe å ta stilling til om loggmeldingene var viktige, eller ikke 
3. Det skjuler seg en litt halv-hacky oppdatering av iverksettingen som benyttes i nyIverksetting - beløp og perioder oppdateres med "siste nytt" 

Tror jeg prøver å lage en versjon som ikke skjuler dette på denne måten.